### PR TITLE
ACS-4811 updates rubocop config README

### PR DESCRIPTION
**Pull request by AndresAbed**
**_JIRA Card_**: https://qcentrix.atlassian.net/browse/ACS-4811
**_Summary and description of changes_**:
Updates the rubocop config README

### CODE REVIEW

File: ruby/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # How to use Rubocop configs
-Reference the rubocop config that you want from another repo by inheriting from this file. When you are ready to upgrade rubocop to a different version, change the file you `inherit_from` in your repo's local `.rubocop.yml`.
+Reference the rubocop config that you want from another repo by inheriting from one of this `.rubocop-...` files.
```
Comment (rebeccafae) [inline ruby/README.md]:
```suggestion
Reference the rubocop config that you want from another repo by inheriting from one of the `.rubocop-...` files.
```

File: ruby/README.md [hunk 2]
```diff
@@ -34,7 +38,5 @@ Note: Anything you add to your local yml file will overwrite what is in the inhe
         ```
           channel: rubocop-0-77
         ```
-2. `.rubocop.yml`
-    1. Update the url you are inheriting from. This should match the url in your `codeclimate.yml`
 3. Make sure your `Gemfile` or `gemspec` specifies the same version of rubocop as the config you are using.
 4. If you are using a new type of cop for the first time (i.e. rspec or grpahql) you may need to add the gems to your gemfile.
```
Comment (rebeccafae) [inline ruby/README.md]:
```suggestion
4. If you are using a new type of cop for the first time (i.e. rubocop-rspec or rubocop-graphql) you may need to add the gems to your gemfile. (These will also be listed under `require` at the top of the yml file) 
```

**Review by rebeccafae — APPROVED** [review]
Thanks for the improvements! I made a couple small suggestions based on our earlier conversations but they are not necessary.

**Review by albertski — APPROVED** [review]
Looks great.

**Pull request merged**